_extensions/odoo/__init__.py

```diff
@@ -14,12 +14,12 @@
 from docutils import nodes
 def setup(app):
     if hasattr(app, 'set_translator'):
-        app.set_translator('html', translator.BootstrapTranslator)
+       app.set_translator('html', translator.BootstrapTranslator)
     else:
-        if getattr(app.config, 'html_translator_class', None):
-            app.warn("Overriding the explicitly set  html_translator_class setting",
-                     location="odoo extension")
-        app.config.html_translator_class = 'odoo.translator.BootstrapTranslator'
+       if getattr(app.config, 'html_translator_class', None):
+           app.warn("Overriding the explicitly set  html_translator_class setting",
+                    location="odoo extension")
+       app.config.html_translator_class = 'odoo.translator.BootstrapTranslator'
```

Comment:
Why did these lines get reindented / unintented?

Reply:
I moved it to #559 to port the changes starting from 11.0 and added the spaces.